exciting_environments/utils.py

```diff
@@ -0,0 +1,7 @@
+import jax_dataclasses as jdc
+
+
+@jdc.pytree_dataclass
+class Normalization:
```

Comment:
Just as a suggestion:

```suggestion
class MinMaxNormalization:
```

To directly differentiate from other normalization techniques, e.g., logarithmic etc. I am not sure if we ever want to add some other form of normalization, but I think it might prevent some confusion.

Reply:
Sounds good to me.